tests/cli/test_cli.py

```diff
@@ -1639,6 +1639,15 @@ def test_status(ctx):
     check_contains("Commands for bundles", cl_output)
     check_equals(cl_output, help_output)
 
+    # Ensure that the "cl help" command doesn't take too long to start up.
```

Comment:
I'm adding a test here to try to ensure this doesn't regress. At least, I hope that if the regular "cl" command suddenly takes a long time to start up, this will cause people to look into it in the future and ensure no large imports are taking up too much time.

Reply:
Actually, removing this test because it seems too flaky (it sometimes takes up to 3.5 seconds in CI)